### PR TITLE
chore: switch from `yarn` to `pnpm`

### LINKED ISSUE #1249: TODO: pnpm に切り替え
本家はだいぶ前に移行しており、Yarn を使い続ける理由もないので手が空いた時に移行したい
`https://github.com/vitejs/vite/pull/5060`

### PR DESCRIPTION
**Pull request by NozomuIkuta**
resolve #1249

This is my very first PR to this repository.
I might be missing something important.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ jobs:
       with:
         node-version: 18
     - name: Install deps
-      run: yarn
+      run: pnpm install
     - name: Run textlint
-      run: yarn lint
+      run: pnpm run lint
```
Comment (NozomuIkuta) [inline .github/workflows/lint.yml:18]: I do not short syntax like `pnpm lint` according to https://github.com/vitejs/vite/pull/5060#discussion_r715237130.

**Comment by NozomuIkuta** [comment]
@jay-es 

Please take over this PR and fixes Netlify configuration to pass CI.

**Comment by jay-es** [comment]
@NozomuIkuta Thank you very much!
I imported your commit into my local repository, fix the `lint` action, and created a new PR #1257. 
Is this the right way?

@kazupon Excuse me.
I don't have access to Netlify, so please invite me.
Or could you change the build command for the site to pnpm.

**Comment by NozomuIkuta** [comment]
Thank you for taking it over.
I will check your PR later.

Let me close this PR for now.

**Comment by kazupon** [comment]
@jay-es netlify ですが、viteの方は自分入っていないので権限周りいじれないです。🙏

**Pull request closed without merging**